### ISSUE TITLE
Hyrax 2.4.1 upgrade

### ISSUE THREAD

**Issue by kerchner** [gwu-libraries/scholarspace-hyrax#158]

**Comment by kerchner** [comment]
Note that this breaks the etd upload script (and likely the batch loader):
```NameError: uninitialized constant ActiveFedora::Noid
/opt/scholarspace/scholarspace-hyrax/lib/tasks/gwss.rake:193:in `ingest_etd'
/opt/scholarspace/scholarspace-hyrax/lib/tasks/gwss.rake:129:in `block (2 levels) in <top (required)>'
/usr/local/rvm/gems/ruby-2.3.3/gems/rake-12.3.2/exe/rake:27:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.3.3/bin/ruby_executable_hooks:15:in `eval'
/usr/local/rvm/gems/ruby-2.3.3/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => gwss:ingest_etd
(See full trace by running task with --trace)
Traceback (most recent call last):
  File "etd_loader.py", line 764, in <module>
    l.import_etds()
  File "etd_loader.py", line 492, in import_etds
    self.store.get(etd_id), self.ingest_depositor)
  File "etd_loader.py", line 620, in repo_import
    output = subprocess.check_output(command, cwd=self.ingest_path)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['rake', 'RAILS_ENV=production', 'gwss:ingest_etd', '--', '--manifest=/tmp/tmp8nofc__u/metadata.json', '--primaryfile=/tmp/tmp8nofc__u/Kirkendoll_gwu_0075A_14554.pdf', '--depositor=openaccess@gwu.edu']' returned non-zero exit status 1```

Likely fix is to swap out `ActiveFedora::Noid` with `Rails::Noid` (??) - I may have seen this in a Hyrax release note but need to re-find it.

Ref https://github.com/samvera/hyrax/pull/2368/files

**Comment by kerchner** [comment]
Fixed in #205
